libats-http/src/main/scala/com/advancedtelematic/libats/http/ServiceHttpClient.scala

```diff
@@ -62,13 +63,21 @@ abstract class ServiceHttpClient(_httpClient: HttpRequest => Future[HttpResponse
   protected def execHttp[T : ClassTag](request: HttpRequest)
                                       (errorHandler: PartialFunction[RemoteServiceError, Future[T]] = defaultErrorHandler())
                                       (implicit um: FromEntityUnmarshaller[T]): Future[T] =
+    execHttp2(request)(errorHandler).map(_.unmarshalledResponse)
```

Comment:
`2` is not really descriptive, I think it should have a name that describes why it's different from just `execHttp`

Reply:
TBH I spent too much time trying to think of a better name but I wasn't quite satisfied with anything I could think of... `execHttpUnmarshalled` didn't seem quite right either because `execHttp` is also unmarshalled... I'm open for suggestions though, I agree this is not great.